### PR TITLE
Customize new templates

### PR DESCRIPTION
**Pull request by nathanbegbie**
search_results.pt, search_box.pt, paginator.pt and 404.pt

### CODE REVIEW

**Comment by nathanbegbie** [comment]
ebola_response___universal_core
ebola_response___universal_core

@SteveBarnett Please review

File: unicorecmsebola/static/css/style.css [hunk 1]
```diff
@@ -76,6 +76,10 @@
 	#menu { background: #d15c4d; color: #f8f7e7; text-align: center; }
 		#menu a { padding: 8px 10px; display: block; border-bottom: 1px solid #fff; text-align: center; color: #f8f7e7; }
 
+/* BASE */
```
Comment (SteveBarnett) [inline unicorecmsebola/static/css/style.css:79]: `SEARCH`?

Comment (nathanbegbie) [inline unicorecmsebola/static/css/style.css:79]: Ahem. Yeah, i think it was the MNM CSS that had that as the descriptor, so I was trying to be consistent. On reflection that's consistently bad.

**Comment by SteveBarnett** [comment]
@nathanbegbie That numbering of the articles makes the search results look confusing:

> "Why are the numbers in the wrong order?".

I know it's more work, but we need a filter to ditch the numbers (in search results).

**Comment by SteveBarnett** [comment]
On the search results page, please add value="${query}" on the search input.

People will often refine a query, or try something similar if they don't find what they were looking for a first pass.

**Comment by nathanbegbie** [comment]
@miltontony review please

**Comment by miltontony** [comment]
@nathanbegbie @SteveBarnett `-1` on fudging the titles. I agree it's confusing to have the numbers out of order, but that's the data that we're working with. Same can be said if we had content that was alphabetical and the search results are shown out of order. I say let's leave it as is.

**Comment by SteveBarnett** [comment]
@miltontony: I disagree with your disagreement :). From a UX standpoint, I can't accept the search results as they are: numerical results displayed out of order.

I don't think this is the same as the alphabetical case. You often see things in non-alphabetical order in interfaces: navigation, category listing, search results, article listings, kind of things. Users (people!) expect numbers to be in number order, though.

So, I think the options are:
- display the results without numbers;
- display them in numerical order.

What do you think?

cc @nathanbegbie

**Comment by miltontony** [comment]
@nathanbegbie so the current hack to just remove the numbers is likely to fail when your results return static pages (e.g About) that doesn't have numbers. Therefore your only other option is to sort the results by title and not by rank.

**Comment by nathanbegbie** [comment]
@miltontony I can only do that by creating a slightly altered or alternative unicore-cms search function, or create a more stringent title-checker function, which would find the first capitalized letter in the string and start from there, or just strip out the numbers in a title. Unfortunately I'm pretty much out of hours for this PO, so could I remove the template filter for now, create a card and then deal with it after killing chameleon?

**Comment by miltontony** [comment]
@nathanbegbie yup, lets create a ticket for it and remove the hack

File: unicorecmsebola/templates/search_box.pt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal" 
+	 metal:define-macro="search_box"
+	 xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+     i18n:domain="unicorecmsebola">	
+	<div class="search-box">
+		<form action='/search/' method='GET' class='form-search'>
+		    <input id="q" name='q' type="text" hint='enter search here..' i18n:attributes="hint">
```
Comment (miltontony) [inline unicorecmsebola/templates/search_box.pt]: @SteveBarnett shouldn't this be `placeholder` instead of `hint`?

Comment (SteveBarnett) [inline unicorecmsebola/templates/search_box.pt]: @miltontony Yeah, `placeholder` is the standard. The placeholder text should be an example, though, and I can't think of something generic that would be useful. I say rather just ditch hint.

Comment (miltontony) [inline unicorecmsebola/templates/search_box.pt]: +1 on ditching `hint` everywhere

**Pull request merged**